### DISCUSSION TITLE
Use of RMA normalised microarray data

### DISCUSSION THREAD

**Discussion by novino11** [omnideconv/immunedeconv#89] [General]
HI,
I'm trying to deconvolute microarray data using the immunedeconv package. I was wondering if I could use log2RMA normalised data for the purpose or not. From the documentation, I assume this can be used for "XCell". However, under the function deconvolute- there is an option to specify arrays= TRUE/ FLASE. I was wondering if this would enable me to choose quantiseq for deconvolution as well with microarray data.

**Comment by grst** [comment]
Hi @novino11, 

xCell and MCP-counter should work with any kind of normalization. 

Quantiseq and CIBERSORT support the `arrays=TRUE` option. However, I'm currently not sure what the recommended normalization for microarray data is. 

@FFinotello, can you help?

**Reply by FFinotello** [reply]
quanTIseq obtained nice results on two microarray datasets, GSE65133 and GSE20300. I do not remember the exact normalization approach used in the original publications, but it should be reported in GEO.

Starting from this normalized data, we processed the data as follows: "Probes were mapped to gene symbols with the biomaRt R package. In case of multiple probes mapping to the same gene symbol, the probe with the highest average expression across all samples was selected."

Nevertheless, I think xCell and MCP-counter would be more suited for this data.
